src/main/java/panav/task/TaskList.java

```diff
@@ -0,0 +1,101 @@
+package panav.task;
+
+import java.util.ArrayList;
+
+/**
+ * Class to represent the list of Tasks in this program. It contains functionality
+ * to edit the task list and add/delete changes.
+ */
+public class TaskList {
+
+    private ArrayList<Task> tasks;
+
+    public TaskList(ArrayList<Task> src) {
+        tasks = new ArrayList<>();
+        for (Task task : src) {
+            tasks.add(task);
+        }
```

Comment:
Great code quality that you have shown, as the add function in ArrayList is very efficient! :D